osquery/tests/integration/tables/os_version.cpp

```diff
@@ -36,4 +36,14 @@ TEST_F(OsVersion, test_sanity) {
   validate_rows(data, row_map);
 }
 
+// Test for install_date in Windows. This should be removed when other OS get
+// this
+TEST_F(OsVersion, test_installdate) {
+  if (isPlatform(PlatformType::TYPE_WINDOWS)) {
+    SQL results("select install_date from os_version");
+    EXPECT_EQ(results.rows().size(), 1U);
+    EXPECT_FALSE(results.rows()[0].at("install_date").empty());
```

Comment:
Would you like to do it in terms of `test_sanity` test? Just add one more item to the map `row_map` if the platform is Windows.

Reply:
Would ValidationMap support an if statement mid definition? Or are you saying I should make it something like:

if (windows)
  //Map with extra row

else
  //Map without row

Reply:
You could do something like in `osquery/tests/integration/tables/interface_addresses.cpp` or just insert one more column below if platform is windows. Whatever you prefer :)